kuma/wiki/models.py

```diff
@@ -1739,6 +1739,9 @@ def content_cleaned(self):
 
     @cached_property
     def previous(self):
+        return self._previous()
+
+    def _previous(self):
```

Comment:
Meh, no private methods please. You can use this:
```python
previous = cached_property(get_previous, name='previous')
```
where `get_previous` would be the new name of the existing method